app/js/views/settings.js

```diff
@@ -26,55 +33,68 @@ export default class SettingsView extends View {
     super(controller);
 
     this.modal  = this.$('gaia-modal');
+    this.header = this.$('gaia-header');
     this.addons = this.$('gaia-list');
 
+    this.closeButton       = this.$('button[data-action="close"]');
+    this.mergeModeButton   = this.$('button[data-action="mergeMode"]');
+    this.doneMergingButton = this.$('button[data-action="doneMerging"]');
+
     this.on('click', 'gaia-button', this._handleClick.bind(this));
     this.on('click', 'button', this._handleClick.bind(this));
+    this.on('click', 'a', this._handleClick.bind(this));
 
-    // this.on('change', 'gaia-switch', this._handleChange.bind(this));
+    this.on('click', 'gaia-checkbox', () => setTimeout(this._handleSelectionChange.bind(this)));
+    this.addons.addEventListener('click', this._handleSelectionChange.bind(this));
```

Comment:
We could do this instead:
1. Make the contents of each add-on record unclickable.
2. Bind a 'click' listener to the outer-most element of each add-on record.
3. Toggle the checkbox and handle the selection from there.